### PR TITLE
Extending documentation

### PR DESCRIPTION
**Pull request by goldmann**

### CODE REVIEW

File: docs/descriptor.adoc [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Dogen descripor file
```
Comment (l-d-j) [inline docs/descriptor.adoc]: typo here probably - missing T

File: docs/descriptor.adoc [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Most important, user facing part of Dogen is the descriptor file. We use YAML format which is easy to read and edit by humans but still machine parseable. By convention we use the `image.yaml` name for.
+
+## General section
+
+This is the general description of the image.
+
+* `name` -- this is the image name without the registry part.
+* `version` -- version of the image
+* `release` -- also called build number. This value should be bumped every time you build the image. It can be string or integer, or mix of these.
+
```
Comment (l-d-j) [inline docs/descriptor.adoc]: missing from instruction from example bellow

File: docs/descriptor.adoc [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    - "/opt/eap/bin/wrapper.sh"
+cmd:
+    - some
+    - argument
+```
+
+## Packages
+
+If you need to install additional packages you can use the `packages` section where you specify package names to be installed.
+
```
Comment (l-d-j) [inline docs/descriptor.adoc]: maybe we should mention its yum only

File: docs/descriptor.adoc (hunk 1 above)
Comment (goldmann) [inline docs/descriptor.adoc]: Will fix it!

File: docs/descriptor.adoc (hunk 2 above)
Comment (goldmann) [inline docs/descriptor.adoc]: Good catch :)

File: docs/descriptor.adoc [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+You can specify the entrypoint or command that should be used by the container with the `cmd` and `entrypoint`.
+
+NOTE: Both `entrypoint` and `cmd` keys use the exec (array) form of providing its value which is preferred in Docker.
+
+```yaml
+entrypoint:
+    - "/opt/eap/bin/wrapper.sh"
+cmd:
+    - some
```
Comment (goldmann) [inline docs/descriptor.adoc]: Can do it, sure!

File: docs/descriptor.adoc (hunk 3 above)
Comment (goldmann) [inline docs/descriptor.adoc]: This is when I realized that we should make it a plugin and I stopped writing anything more there.

File: docs/descriptor.adoc [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+Most important, user facing part of Dogen is the descriptor file. We use YAML format which is easy to read and edit by humans but still machine parseable. By convention we use the `image.yaml` name for.
+
+## General section
+
+This is the general description of the image.
+
+* `name` -- this is the image name without the registry part.
+* `version` -- version of the image.
+* `release` -- also called build number. This value should be bumped every time you build the image. It can be string or integer, or mix of these.
+* `from` -- yu specify the base image for your image.
```
Comment (l-d-j) [inline docs/descriptor.adoc]: yu -> you :)

**Comment by goldmann** [comment]
Updated!

**Pull request merged**